packages/manager/apps/container/src/container/nav-reshuffle/onboarding/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState, useMemo } from 'react';
+
+import useOnboarding from '@/core/onboarding';
+import useProductNavReshuffle from '@/core/product-nav-reshuffle';
+
+import OnboardingIntroduction from './introduction';
+import OnboardingWalkMe from './walkMe';
+
+export const NavReshuffleOnboardingWidget = () => {
+  const onboarding = useOnboarding();
+  const productNavReshuffle = useProductNavReshuffle();
+
+  const [isWalkMeVisible, setIsWalkMeVisible] = useState();
+
+  useMemo(() => {
+    setIsWalkMeVisible(
+      onboarding.hasStarted(productNavReshuffle.onboardingOpenedState),
+    );
+  }, [productNavReshuffle.onboardingOpenedState]);
+
+  return (
+    <>
```

Comment:
Empty tag ?

Reply:
Yes, React expect a single parent node at render

Reply:
React expects to have a single parent element, so here we use the short syntax for React Fragments to avoid to add extra nodes to the DOM.
(https://en.reactjs.org/docs/fragments.html)